indico/modules/events/fields.py

```diff
@@ -112,6 +112,14 @@ def can_enter_manually(self):
             return True
         return self.event.can_manage(session.user) or not persons_settings.get(self.event, 'disallow_custom_persons')
 
+    @property
+    def name_format(self):
+        from indico.modules.users.models.users import NameFormat
+        name_format = layout_settings.get(self.event, 'name_format') if self.event else None
+        if name_format is None and session.user:
+            name_format = NameFormat(session.user.settings.get('name_format'))
```

Comment:
for some reason the `EnumConverter` in the user settings is not working... Couldn't figure out why :/